src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,32 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMAL_AGE = 18;
+    private static final int MINIMAL_PASSWORD_LENGTH = 6;
+    private StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getLogin() == null || user.getLogin().isEmpty() || user.getLogin().isBlank()
+                || user.getAge() == null) {
+            throw new RuntimeException("Please, enter a valid login");
+        } else if (storageDao.get(user.getLogin()) != null) {
```

Comment:
```suggestion
        } 
        if (storageDao.get(user.getLogin()) != null) {
```
else блок можна не використовувати якщо перед цим є throw або. return